include/kernel/sporadic.h

```diff
@@ -141,14 +141,10 @@ void refill_update(sched_context_t *sc, ticks_t new_period, ticks_t new_budget,
 
 
 /* Charge `usage` to the current scheduling context.
- * This function should only be called only when charging `used` will deplete
```

Comment:
is this no longer true??

Reply:
I think this still holds.

Reply:
Can you put the comment back then please

Reply:
Oh, I didn't realise it had been removed.